PL2/PL2-core/src/test/java/nl/tudelft/pl2016gr2/core/algorithms/FilterSnipsTest.java

```diff
@@ -27,7 +27,7 @@
   @Before
   public void setup() {
     for (int i = 1; i <= 9; i++) {
-      Node node = new Node(i, 1, null, 0);
+      Node node = new Node(i, 1, new ArrayList<>(), 0);
       if (i == 1 || i == 4 || i == 7) {
```

Comment:
Dat is code van casper, heb alleen deze test aangepast omdat die faalde door null.
`i % 3 == 1` lijkt me handiger